godtools/App/Services/Renderer/ManifestResourcesCache/ManifestResourcesCache.swift

```diff
@@ -7,25 +7,26 @@
 //
 
 import UIKit
+import GodToolsToolParser
 
-class ManifestResourcesCache: ManifestResourcesCacheType {
+class ManifestResourcesCache {
     
-    private let manifest: MobileContentManifestType
+    private let manifest: Manifest
     private let translationsFileCache: TranslationsFileCache
     
-    required init(manifest: MobileContentManifestType, translationsFileCache: TranslationsFileCache) {
+    required init(manifest: Manifest, translationsFileCache: TranslationsFileCache) {
         
         self.manifest = manifest
         self.translationsFileCache = translationsFileCache
     }
     
     private func getSHA256FileLocation(fileName: String) -> SHA256FileLocation? {
         
-        guard let resource = manifest.getResource(fileName: fileName) else {
+        guard let localeName = manifest.resources[fileName]?.localName else {
```

Comment:
there is still a `manifest.getResource(fileName)` method that will give you the appropriate resource

Reply:
either way works.

I would have wanted to eventually make the actual `resources` Dictionary internal to the library, but with the upcoming tool distribution changes, we will need to be able to access all the resources that are included with a tool when downloading it

Reply:
scratch this, that method is internal only. I guess the normal expected usage is to access the resource object directly from the Image/Animation/etc that is referencing it.